src/Themes/basic-theme.scss

```diff
@@ -66,9 +66,9 @@
   --link-on-secondary-color: var(--color-link_color);
 
   // Shadows
-  --box-shadow-small: 0px 4px 8px rgba(0, 0, 0, 0.2);
-  --box-shadow-medium: 0px 6px 20px rgba(0, 0, 0, 0.2);
-  --box-shadow-large: 0px 15px 50px rgba(0, 0, 0, 0.3);
+  --box-shadow-small: #{$shadow-size-small} rgba(0, 0, 0, #{$shadow-small-opacity});
+  --box-shadow-medium: #{$shadow-size-medium} rgba(0, 0, 0, #{$shadow-medium-opacity});
+  --box-shadow-large: #{$shadow-size-large} rgba(0, 0, 0, #{$shadow-large-opacity});
```

Comment:
why do you prefer splitting to two SASS vars(`$shadow-size-small`, `$shadow-small-opacity`) over a single SASS var that will contain all the definition? e.g.`$shadow-size-small: 0px 4px 8px rgba(0, 0, 0, 0.2);`

Reply:
because its a function with 3 parameters.
we want to be able to control them separately and choose only what is mutating. 
This enable readability and allow understanding of the context for each change.